src/components/FileDetail/FileDetail.js

```diff
@@ -58,61 +57,70 @@ const FileDetail = ({fileNode, history, match}) => {
                       All Documents
                     </Button>
                   </Link>
-                </Grid.Column>
-              </Grid.Row>
-              <Grid.Row>
-                <Grid.Column mobile={16} tablet={16} computer={12}>
                   <Header as="h3" size="medium" color="blue">
                     {fileNode.name}
                   </Header>
                 </Grid.Column>
-                <Grid.Column mobile={16} tablet={16} computer={4}>
-                  <Button.Group size="mini">
-                    <Button
-                      basic
-                      icon="pencil"
-                      onClick={() => setDialog('annotation')}
-                      data-testid="edit-button"
-                      content="EDIT"
-                    />
-                    <Button
-                      primary
-                      icon="download"
-                      onClick={e =>
-                        downloadFile(
-                          studyId,
-                          fileNode.kfId,
-                          null,
-                          downloadFileMutation,
-                        )
-                      }
-                      content="DOWNLOAD"
-                    />
-                    <Popup
-                      trigger={<Button basic icon="trash alternate" />}
-                      header="Are you sure?"
-                      content={
-                        <>
-                          This file and all of its versions and history will be
-                          deleted
-                          <Divider />
-                          <Button
-                            data-testid="delete-confirm"
-                            negative
-                            fluid
-                            icon={<Icon name="trash alternate" />}
-                            content="Delete"
-                            onClick={e => {
-                              deleteFile({variables: {kfId: fileNode.kfId}});
-                              history.goBack();
-                            }}
-                          />
-                        </>
-                      }
-                      on="click"
-                      position="top right"
-                    />
-                  </Button.Group>
+                <Grid.Column mobile={16} tablet={16} computer={3}>
```

Comment:
It would be nice to have some gutters between the buttons as they are now, or use button groups to make them fit together more cleanly if that's what is wanted like the below:
<img width="235" alt="Screen Shot 2019-07-18 at 9 39 44 AM" src="https://user-images.githubusercontent.com/2495894/61462363-7fe18d00-a940-11e9-97e5-a2d7b731891b.png">